version/mkversion/mkversion.go

```diff
@@ -199,6 +199,25 @@ func tailscaleModuleRef(modBs []byte) (string, error) {
 }
 
 func mkOutput(v verInfo) (VersionInfo, error) {
+	if override := os.Getenv("TS_VERSION_OVERRIDE"); override != "" {
```

Comment:
I think you can replace all this with the function `parseVersion` that is already in this file. Then you can do

```
v.major, v.minor, v.patch, err = parseVersion(override)
if err != nil { ...
```

Reply:
ohh, not sure how I missed that function :facepalm: 
done!